src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -63,6 +63,12 @@ Namespace Microsoft.CodeAnalysis
             Return nodeOrToken.RawKind = kind
         End Function
 
+        ''' <inheritdoc cref="SyntaxNode.ContainsDirective"/>
+        <Extension>
+        Public Function ContainsDirective(node As SyntaxNode, kind As SyntaxKind) As Boolean
+            Return node.ContainsDirective(CType(kind, Integer))
```

Comment:
Prefer `DirectCast` over `CType`, even though `CType` is used in other parts of this file.

Reply:
actually, i think CInt is most idiomatic here, right?

Reply:
I'm not 100% sure. @AlekseyTs?

Reply:
can't DirectCast here (VB doesn't allow it).  I switched to CInteger.